src/System.Security.Cryptography.Pkcs/tests/EnvelopedCms/DecryptTests.cs

```diff
@@ -117,6 +117,13 @@ public static void Decrypt_512_NoData_FixedValue()
                 "650304012A0410280AC7A629BFC9FD6FB24F8A42F094B4"
             ).HexToByteArray();
 
+            if (PlatformDetection.IsFullFramework)
+            {
+                // On NetFx when Array.Empty should be returned an array of 6 zeros is
+                // returned instead.
```

Comment:
Ummm... :smile:

Reply:
Yep, it's special.  First call gives ERROR_MORE_DATA (6), second call says cbResult was 0, but it didn't check it.